TEXTBOOK_README.md

```diff
@@ -0,0 +1,113 @@
+
+
+## Steps to participate in Qiskit textbook translations:
+
+1. Open the [TEXTBOOK_CONTRIBUTORS](https://github.com/qiskit-community/qiskit-translations/TEXTBOOK_CONTRIBUTORS) file. Look for the language header that you'd like to contribute to and add your full name under he header. If you do not find the language that you'd like to contribute to, please read [this section](#how-to-add-a-new-language).<br/>
+2. Create a pull request (PR) to add your name to the list. Make sure to follow the template to open a PR.<br/>
+Each contributor must create their own PR and sign the Contributors License Agreement (CLA). (see #3 below).
+If you have an open issue for a language request, **add the issue link to the PR**.
+
+   _(Not sure how to open a PR? Checkout [this section](#how-to-add-my-name-to-textbook_contributors-list-and-open-a-pr-in-github))_
+   
+3. When you conribute to a Qiskit open source project on GitHub with a new pull request, a bot will evaluate whether you have signed the [Qiksit Contribution License Agreement (CLA)](https://qiskit.org/license/qiskit-cla.pdf). If it is your first time contributing to Qiskit, you will be prompted to sign the CLA in your PR. Please make sure to sign the CLA by accepting the agreement.<br/>
+4. Join [GitLocalize](https://gitlocalize.com) using the same GitHub ID you used to open your PR above.  
+5. Once your PR has been accepted and merged, you will receive a notfication from GitLocalize with a title: [GitLocalize]You have been assined a new role.<br/>
+6. Join project [Qiskit/platypus] and then visit the language you signed up to contribute to.<br/>
+7. For questions about the project, to connect with other translators, or to receive updates regarding the project, please join #qiskit-localization channel in the [Qiskit workspace](http://ibm.co/joinqiskitslack). <br/>
+
+## Review the CLA document
+The [Qiksit Contribution License Agreement (CLA)](https://qiskit.org/license/qiskit-cla.pdf) document is available for review as PDF.
+
+## What is GitLocalize?
+
+[GitLocalize](https://gitlocalize.com) is a platform for managing translations of the content stored on GitHub and simplifies the translation workflow for developers.
+
+Important note:<br/>
+Gitlocalize is under active development, so you may encounter issues during translation. If you find bugs or errors, please feel free to reach out to translation leads or open an issue [here](https://github.com/gitlocalize/gitlocalize-ibm/issues). 
+
+
+## How to add a new language?
+
+The Qiskit Localization Project currently supports: Bengali, Chinese Simplified, Chinese Traditional, French, German, Greek, Hindi, Hungarian, Indonesian, Italian, Japanese, Korean, Malayalam, Marathi, Polish, Portuguese, Russian, Spanish, Swahili, Tamil, Thai, Turkish, Urdu, and Vietnamese. 
```

Comment:
Do we wanna support all 24 languages together?

Reply:
Yes, I want to encourage existing translation leads to continue their efforts with the textbook. What do you think?